Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -0,0 +1,1436 @@
+    Otherwise (if it is set to 0 or not exists) it says that it is an attribute's tag.
+    If the data is event's (is_event_level = true) we would like to add to tag_set_ids all the tags
+    (event ones and the event's attribute tags ones as it is part of the event scope).
+    If the data is attribute's (is_event_level = false), and the tag is only related to an attribute
+    we would like to add it to tag_set_ids. In any other case, we won't add the tag.
+
+    Args:
+        attribute_dict (dict): The dictionary that includes the tag list.
+        is_event_level (bool): Whether the attribute_dict was received from an event object,
+        meaning the tags are event's ones. Otherwise, the data is attribute's (attribute tags).
+    """
+    output = []
+    tag_set_ids = set()
+    tags_list = attribute_dict.get('Tag', [])
+    for tag in tags_list:
+        is_event_tag = tag.get('inherited', 0)  # field doesn't exist when this is an attribute level, default is '0'
+        tag_id = tag.get('id')
+        if is_event_level:
+            tag_set_ids.add(tag_id)
+        else:  # attribute level
+            if not is_event_tag:
+                tag_set_ids.add(tag_id)
+        output.append({'ID': tag_id, 'Name': tag.get('name')})
+    return output, tag_set_ids
+
+
+def parse_response_reputation_command(response, malicious_tag_ids, suspicious_tag_ids):
+    """
+    Todo : change this , not correct anymore!
+    After getting all the attributes which match the required indicator value, this function parses the response.
+    This function combines all the attributes that found with their event's object into one json object (that will
+    be returned to the context data), this is the reason why we use the first attribute object.
+    We modify the first_attribute object to include the full information about the related events that were found.
+    In addition the function returns the indicator score and the tag (id) which made the indicator to get that score.
+    Please see an example for a response in test_data/reputation_command_response.json
+    Please see an example for a parsed output in test_data/reputation_command_outputs.json
+        """
+    attributes_list = response.get('Attribute')
+    if not attributes_list:
+        return None
+    found_related_events, attributes_tag_ids, event_tag_ids = prepare_attributes_array_to_context_data(response)
+    is_attribute_in_event_with_bad_threat_level_id = found_event_with_bad_threat_level_id(found_related_events)
+    score, found_tag = get_score(attribute_tags_ids=attributes_tag_ids, event_tags_ids=event_tag_ids,
+                                 malicious_tag_ids=malicious_tag_ids, suspicious_tag_ids=suspicious_tag_ids,
+                                 is_attribute_in_event_with_bad_threat_level_id=
+                                 is_attribute_in_event_with_bad_threat_level_id)
+
+    response = replace_keys_from_misp_to_context_data(response)  # this is the outputs (Attribute)
```

Comment:
if I am not mistaken, replace_keys_from_misp_to_context_data creates a new object, so it should not overwrite `response` but to create a new variable.
maybe `formatted_response`?